.jshintrc

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
should take jshintrc from here https://github.com/aldryn/aldryn-boilerplate-bootstrap3/blob/develop/.jshintrc